api/controllers/payment_providers.py

```diff
@@ -8,7 +8,7 @@
 from api.schemas import PaymentUpdate, PaymentCreate
 
 
-@app.post("/payment/add")
+@app.post("/payments/add")
```

Comment:
I was going to say something, but it looks like you've already changed all instances of `PaymentProider` to `PaymentProvider`...